source/_layouts/post.blade.php

```diff
@@ -32,13 +32,13 @@
                       <img src="https://www.gravatar.com/avatar/{{$page->gravatar}}?size=40"
                       alt="{{ $page->author }}" />
                       <span>
-                        By <a href="{{ $page->baseUrl }}team/{{ \Illuminate\Support\Str::slug($page->author) }}"> {{ $page->author }} </a>
+                        {{ $page->t("By") }} <a href="{{ $page->baseUrl }}team/{{ \Illuminate\Support\Str::slug($page->author) }}"> {{ $page->author }} </a>
                       </span>
                     </div>
 
                     <div class="ud-blog-meta">
                       <p class="date">
-                        <i class="lni lni-calendar"></i> <span>{{ date('F j, Y', $page->date) }}</span>
+                        <i class="lni lni-calendar"></i> <span>{{ $page->t(date('F',$page->date)) . " ". date('j, Y', $page->date) }}</span>
```

Comment:
This don't will solve the problem because we could have a different date format in a specific language.

A possible way to solve this is this package and setting the locale:

https://github.com/CarbonPHP/carbon

I think that have a possible solution here:

https://carbon.nesbot.com/docs/#api-localization